intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -751,10 +752,35 @@ \subsection{Setting per-member VO quota}
         100\% can actually be used.
 \end{enumerate}
 
-\begin{figure}[h]
+\begin{figure}[!htbp]
   \caption{Setting per-member quota.}
   \centering
     \includegraphics[width=\textwidth]{ch6-VO-per-member-storage}
-\end{figure}
+\end{figure}\label{fig:VO-per-member-storage}
+
+\subsection{VO directories}
+\label{subsec:vo-directories}
+
+When you're a member of a VO, there will be some additional filesystems available:
```

Comment:
Not additional filesystems, but additional directories on each of the shared filesystems (relevant: #158)